composer.json

```diff
@@ -21,7 +21,7 @@
   "minimum-stability": "dev",
   "prefer-stable": true,
   "require": {
-    "php": "^7.3 || ^8.0",
+    "php": "^7.4 || ^8.0",
```

Comment:
Is this necessary?

Reply:
1. Typed property appeared in php 7.4
2. Do we still support 7.3 in BC?

Reply:
1. We are not dependent on type properties, are we? type arguments iirc, and nullable ones are supported from 7.1
2. No, but this repo is public, if not necessary I see no reason to upgrade the minimum required version, wdyt?

Reply:
1. https://github.com/bigcommerce/injector/pull/16/files#diff-9bf09a09806ea01fdfb744f558f7006011d5a81fb37806e82d4509d007596cf6R11 we can change it.
2. https://www.php.net/supported-versions.php I recommend supporting at least a stable version of the PHP.